### DISCUSSION TITLE
Graphics in QZ app

### DISCUSSION THREAD

**Discussion by juanrancho** [cagnulein/qdomyos-zwift#2275] [General]
Hi,
I am a very very lover of your app.
As I said in other discussion, this is what I am waiting for too much longer (I train indoor in my own home-gym and I use my smart bike (zBike:  https://zycle.eu/zbike/) and my Threadmill (Nordictrack T10.0: https://www.fitnessdigital.com/cinta-de-correr-nordictrack-t10.0-1-mes-de-suscripcion-ifit-individual/p/10009946/)
I am testing the app (I have installed the beta version in my android phone - Samsung zFolder 5) and I have a problem or perhaps a dobut:
- After I have done a trainning session (HR rate controlled automatically) and I press the stop buttom, the graphics that I can view are completelly empty (I am attaching them), but if I import the .fit file to Garmin Connect, there are all the information.
What could be the reason?
Can I fix it?

![garmin connect QZ running session 2024-04-04 JCRP](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/6945c02c-1e8f-4b2d-bdcf-e0efcbe0840c)
![QZ running session 2024-04-04 JCRP (1)](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/4dfdc46c-8ba3-4ba1-956f-751f4c33a028)
![QZ running session 2024-04-04 JCRP (1)](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/d68032e1-e9ec-42a8-9945-d53ca97ca371)
![QZ running session 2024-04-04 JCRP (2)](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/15144ce2-d254-435c-8c58-30978956e178)
![QZ running session 2024-04-04 JCRP (3)](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/808160e7-eec6-41e7-b20b-3d1f04c219f9)
![QZ running session 2024-04-04 JCRP (4)](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/bad6a47f-4dd0-438b-968c-600e7a21f9c7)
![QZ running session 2024-04-04 JCRP (5)](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/649c6d4c-8c24-4028-8b66-15918dcd3700)
![QZ running session 2024-04-04 JCRP (6)](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/f23cfd45-ce52-4565-bf67-3335f4a877f3)
![QZ running session 2024-04-04 JCRP (7)](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/d28f0c24-5d5e-4a38-b254-5adc9352bd3c)
![QZ running session 2024-04-04 JCRP (8)](https://github.com/cagnulein/qdomyos-zwift/assets/163849058/642b3a7f-dc1c-4bde-8755-0dcf9d2c3170)

**Comment by cagnulein** [comment]
hi, i guess that's becaue you didn't hit play in the first place. when you start a workout the red play button MUST NOT be flashing :)
Let me know

**Reply by juanrancho** [reply]
When I start a workout, the threadmill (If I am not wrong) must be started a minimum speed, and then,what is the secuence?

**Reply by cagnulein** [reply]
it depends from your treadmill but the flow is:

- if it's the play button flashing press it, otherwise just run

**Reply by juanrancho** [reply]
ok, I´ll try it , thanks
